src/main/java/seedu/duke/controllers/ModuleServiceController.java

```diff
@@ -54,34 +53,40 @@ public static void chooseToAddToSchedule(Student student, ArrayList<String> sche
             userInput = in.nextLine();
         }
 
-        if (userInput.equals("Y")) {
-            displayMessage("Do you want to keep your completion statuses?");
-            displayMessage("Please input 'Y' or 'N'");
+        //        if (userInput.equals("Y")) {
+        //            displayMessage("Do you want to keep your completion statuses?");
+        //            displayMessage("Please input 'Y' or 'N'");
+        //
+        //            String userInputForCompletion = in.nextLine();
+        //
+        //            while (!userInputForCompletion.equals("N") && !userInputForCompletion.equals(("Y"))) {
+        //                displayMessage("Invalid input, please choose Y/N");
+        //                userInputForCompletion = in.nextLine();
+        //            }
+        //
+        //            displayMessage("Hold on, this may take a while......");
+        //
+        //            Boolean keep;
+        //            if (userInputForCompletion.equals("Y")) {
+        //                keep = true;
+        //            } else {
+        //                keep = false;
+        //            }
+        //            student.getSchedule().addRecommendedScheduleListToSchedule(scheduleToAdd, keep);
+        //            displayMessage("Here is your schedule planner!");
+        //            student.getSchedule().printMainModuleList();
+        //            displayMessage("Happy degree planning!");
+        //
+        //
+        //        } else {
+        //            displayHelp();
+        //        }
```

Comment:
not exactly about code quality but you could do
`/* `
`unused code`
`*/ `
Just need to make sure the indentation of /* and */ is as required